var/spack/repos/builtin/packages/raja/package.py

```diff
@@ -6,33 +6,67 @@
 import socket
 
 from spack.package import *
-from spack.pkg.builtin.camp import hip_repair_cache
+
+from .blt import llnl_link_helpers
+
+
+# Starting with 2022.03.0, the only submodule we want to fetch is tpl/desul
+# since there is no package for it. Other RAJA submodules are defined as
+# dependencies.
```

Comment:
I'm not holding this up for this, but we clearly need to make a desul package available.  Adding it to my list, thanks for pointing out the lack.